modules/enet/doc_classes/ENetPacketPeer.xml

```diff
@@ -18,6 +18,18 @@
 				Returns the number of channels allocated for communication with peer.
 			</description>
 		</method>
+		<method name="get_remote_address" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the IP address of the given peer.
```

Comment:
We normally use "given" to refer to an argument. Here it's the class itself so "of this peer" would likely be better. Same below.